colcon_bundle/verb/utilities.py

```diff
@@ -22,18 +51,17 @@ def update_shebang(path):
     else if python is invoked with parameters
 
     :param path: Path to file to replace shebang in
-    :return: None
     """
     # TODO: We should handle scripts that have parameters in the shebang
     # TODO: We should hangle scripts that are doing other /usr/bin executables
-    py3_shebang_regex = re.compile('#!\s*/usr/bin/python3')
-    py_shebang_regex = re.compile('#!\s*/usr/bin/python')
-    shebang_with_parameters_regex = re.compile('#!\s*/usr/bin/python\s(.+)')
+    py3_shebang_regex = re.compile(r'#!\s*.+python3')
```

Comment:
We should probably look at shebang at the beginning of lines:
`re.compile(r'^#!\s*.+python3')`